custom_components/tahoma/light.py

```diff
@@ -68,13 +85,22 @@ def supported_features(self) -> int:
         if "wink" in self.tahoma_device.command_definitions:
             supported_features |= SUPPORT_EFFECT
 
+        if "setRGB" in self.tahoma_device.command_definitions:
+            supported_features |= SUPPORT_COLOR
+
         return supported_features
 
     def turn_on(self, **kwargs) -> None:
         """Turn the light on."""
         self._state = True
         self._skip_update = True
 
+        _LOGGER.debug(f"light.turn_on kwargs: {kwargs}")
+
+        if ATTR_RGB_COLOR in kwargs:
+            self._rgb = [int(float(c)) for c in kwargs[ATTR_RGB_COLOR]]
```

Comment:
Could you log self._rgb? you need 3 params in `setRGB`, I believe this could be the culprit

Reply:
done.
but if kwargs[ATTR_RGB_COLOR] is a 3 elements list, which the code suggests, then that should work: `self.apply_action("setRGB", *self._rgb)`